src/config/midi.rs

```diff
@@ -15,51 +15,23 @@ impl Midi {
 
 #[derive(Copy, Clone, Debug, Deserialize)]
 pub struct MidiSlider {
-    pub channel: Channel,
-    pub control_change: ControlChange,
-}
-
-#[derive(Copy, Clone, Debug, Deserialize)]
-#[serde(transparent)]
-pub struct Channel {
```

Comment:
These wrapper structs weren't really necessary. Instead I can just tell serde how to deserialize the `wmidi` types:

```
#[serde(deserialize_with = "deserialize_channel")]
```